staging/src/k8s.io/client-go/kubernetes/typed/core/v1/configmap.go

```diff
@@ -97,6 +101,28 @@ func (c *configMaps) List(ctx context.Context, opts metav1.ListOptions) (result
 	return
 }
 
+func (c* configMaps) watchList(ctx context.Context, opts metav1.ListOptions) (result *v1.ConfigMapList, err error) {
+	if opts.ResourceVersionMatch == "" {
```

Comment:
to avoid regenerating the client in the future we could push defaulting to the rest client.